extension/oidcextension/testdata/config.yaml

```diff
@@ -0,0 +1,26 @@
+extensions:
+  oidc:
```

Comment:
Could you leave a comment stating that this would fail, because it lacks the required properties?